src/App.js

```diff
@@ -1,55 +1,82 @@
-import { useEffect, useState } from "react";
+import {useEffect, useState} from "react";
 import './App.css';
 import logo from './mlh-prep.png'
+import city from "city";
```

Comment:
Code improvement Suggestion: You can use single quotes instead of double quotes to make it more aligned with other imports.